hdmap/hdmap_shacl.ttl

```diff
@@ -124,143 +126,7 @@ hdmap:GeneralShape a sh:NodeShape ;
-            sh:maxCount 1 ;
-            sh:order 2 ;
-            sh:path hdmap:xMax ] ;
-    sh:targetClass hdmap:BoundingBox .
-
-hdmap:Coordinate2DShape a sh:NodeShape ;
-    sh:property [ sh:datatype xsd:float ;
-            sh:message "Validation of x failed!"@en ;
-            sh:name "x"@en ;
-            sh:maxCount 1 ;
-            sh:order 0 ;
-            sh:path hdmap:x ],
-        [ sh:datatype xsd:float ;
-            sh:message "Validation of y failed!"@en ;
-            sh:name "y"@en ;
-            sh:maxCount 1 ;
-            sh:order 1 ;
-            sh:path hdmap:y ] ;
-    sh:targetClass hdmap:Coordinate2D .
-    
-hdmap:ProjectLocationShape a sh:NodeShape ;
-    sh:property [ skos:example "nan" ;
-            sh:datatype xsd:string ;
-            sh:description "State of project area"@en ;
-            sh:message "Validation of state failed!"@en ;
-            sh:maxCount 1 ;
-            sh:name "state"@en ;
-            sh:order 1 ;
-            sh:path hdmap:state ],
-        [ skos:example "nan" ;
-            sh:datatype xsd:string ;
-            sh:description "nan"@en ;
-            sh:message "Validation of city failed!"@en ;
-            sh:maxCount 1 ;
-            sh:name "city"@en ;
-            sh:order 3 ;
-            sh:path hdmap:city ],
-        [ skos:example "nan" ;
-            sh:datatype xsd:string ;
-            sh:description "Region of project area"@en ;
-            sh:message "Validation of region failed!"@en ;
-            sh:maxCount 1 ;
-            sh:name "region"@en ;
-            sh:order 2 ;
-            sh:path hdmap:region ],
-        [ skos:example "Germany, GER" ;
-            sh:datatype xsd:string ;
-            sh:description "Country code as ISO 3166-1, alpha-2; "@en ;
-            sh:maxCount 1 ;
-            sh:message "Validation of country failed!"@en ;
-            sh:name "country"@en ;
-            sh:order 0 ;
-            sh:path hdmap:country ;
-            sh:pattern "^[a-zA-Z]{2}-(?:[a-zA-Z]{1,3}|[0-9]{1,3})$" ],
-        [ skos:example "Lat: 1.234- 1.345; Lon 45.23-45.34" ;
-            sh:description "Bounding box with lat/lon values in WGS84"@en ;
-            sh:message "Validation of boundingBox failed!"@en ;
-            sh:name "boundingBox"@en ;
-            sh:node hdmap:BoundingBoxShape ;
-            sh:minCount 1 ;
-            sh:maxCount 1 ;
-            sh:order 5 ;
-            sh:path hdmap:boundingBox ],
-        [ skos:example "Autobahn A8, Großraum Ingolstadt" ;
-            sh:datatype xsd:string ;
-            sh:description "Description of the location, dependen if it is a more trajectory style data or area style data"@en ;
-            sh:message "Validation of relationOrArea failed!"@en ;
-            sh:maxCount 1 ;
-            sh:name "relationOrArea"@en ;
-            sh:order 4 ;
-            sh:path hdmap:relationOrArea ] ;
-    sh:targetClass hdmap:ProjectLocation .     
+    sh:targetClass hdmap:General .   
```

Comment:
is this `GeneralShape` different from the `GeneralShape` in `general`?

Reply:
yes is different
Todo - better naming

Reply:
remove local general node. done!